### PR TITLE
[IMP] hr_*: standardise UI displaying of employees accross all hr apps by using the widget avatar_user

### PR DESCRIPTION
**Pull request by dasz-odoo**
Appraisal: done, fix  chat widget issue
Employee: done, removed unused view
Attendance: done
Expense: done
Time Off: done
Fleet: done
Payroll: done
Referall: done
Lunch: done
https://www.odoo.com/web#id=3132728&menu_id=4720&cids=1&action=333&active_id=1251&model=project.task&view_type=form

### CODE REVIEW

File: addons/hr/views/hr_employee_public_views.xml [hunk 1]
```diff
@@ -46,9 +46,15 @@
                         <div class="row justify-content-between position-relative w-100 m-0 mb-2">
                             <div class="oe_title ps-0 pe-2">
                                 <label for="name" string="Employee Name"/>
+<<<<<<< HEAD
```
Comment (KangOl) [inline addons/hr/views/hr_employee_public_views.xml:49]: Conflict marker committed.

File: training [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/odoo/src/training/
```
Comment (KangOl) [inline training:1]: Not sure this file should be committed.

**Review by Bertrand2 — CHANGES_REQUESTED** [review]
Don't forget to add the id of the task in your commit :) Nice work

File: addons/hr/views/hr_employee_public_views.xml [hunk 3]
```diff
@@ -68,11 +68,18 @@
...
-                                <field name="department_id" context="{'open_employees_kanban': 1}"/>
-                                <field name="employee_type"/>
-                                <field name="company_id" groups="base.group_multi_company"/>
-                                <field name="parent_id"/>
-                                <field name="coach_id"/>
+                                <group>
+                                    <field name="mobile_phone" widget="phone" options="{'enable_sms': false}"/>
+                                    <field name="work_phone" widget="phone" options="{'enable_sms': false}"/>
+                                    <field name="work_email" widget="email"/>
+                                </group>
```
[the remark this answers is not in the document]
Comment (dasz-odoo) [reply addons/hr/views/hr_employee_public_views.xml:75]: nope

File: addons/hr_attendance/views/hr_attendance_view.xml [hunk 4]
```diff
@@ -146,13 +147,37 @@
...
-    <menuitem id="menu_hr_attendance_attendances_overview" name="Attendances" parent="menu_hr_attendance_root" sequence="1" groups="hr_attendance.group_hr_attendance" action="hr_attendance_action_overview"/>
-
-    <menuitem id="menu_hr_attendance_kiosk_no_user_mode" name="Kiosk Mode" parent="menu_hr_attendance_root" sequence="10" groups="hr_attendance.group_hr_attendance_kiosk" action="hr_attendance_action_kiosk_mode"/>
-
-    <menuitem id="menu_hr_attendance_view_attendances" name="Attendances" parent="menu_hr_attendance_root" sequence="10" groups="hr_attendance.group_hr_attendance_user" action="hr_attendance_action"/>
+    <menuitem id="menu_hr_attendance_root"
+              name="Attendances"
+              sequence="205"
+              groups="hr_attendance.group_hr_attendance,hr_attendance.group_hr_attendance_kiosk"
+              web_icon="hr_attendance,static/description/icon.svg"/>
```
[the remark this answers is not in the document]
Comment (dasz-odoo) [reply addons/hr_attendance/views/hr_attendance_view.xml:154]: ok i'll keep them in a single line

**Comment by kba-odoo** [comment]
robodoo r+

**Comment by kba-odoo** [comment]
robodoo r+

**Comment by kba-odoo** [comment]
@robodoo r+

**Comment by kba-odoo** [comment]
@robodoo r+

**Pull request closed without merging**